### ISSUE TITLE
Come up with Accessibility Compliance List

### ISSUE THREAD

**Issue by johnr54321** [hackforla/311-data#443]
# Description
Currently, our MVP was built without any accessibility in mind.  Here is a link to what governments require for a site to be accessible. https://www.w3.org/TR/WCAG20/.  Please come up with a list of all the changes we would need to make and put it in google sheets on our google drive so we can look at what we need to get done.

## Action Items

- [ ] Review Accessibility Requirements
- [ ] Test Site and Draft Corrections

**Comment by johnr54321** [comment]
Spoke with @greenhacks and ETA is 3/24 for initial list.

**Comment by greenhacks** [comment]
@johnr54321 @itsjoycelee Testing is complete; the initial list is on the Google Drive (will Slack you). There were a few scenarios that I was unsure how to test or that required more extensive knowledge of HTML/CSS, so those are marked on the document.
